apps/asap-cli/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "composite": true,
```

Comment:
why do we need it?

Reply:
The "build-typecheck" step, generates .d.ts files for non-composite projects.

These types are then used in the "build sls package" and "build frontend".

asap-cli for example, doesn't need it's types build but it does need to be "typecheck-ed". Which is now done in the "typecheck" step.